src/Cake.Common.Tests/Properties/Resources.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Cake.Common.Tests.Properties {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
```

Comment:
Could you rollback those unrelated changes to `Resources.Designer.cs` leaving only the part that is needed for your test (`Solution_WithProjectsAndFoldersAndMissingLine`)?